.github/workflows/deploy.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: "14"
+          node-version: "16.10.0"
```

Comment:
https://github.com/Qiskit/qiskit.org/pull/2919 makes now possible to set all 16 versions without the need to specify 16.10.

Reply:
Thanks @eddybrando - resolved w/ https://github.com/Qiskit/qiskit.org/pull/2912/commits/c085b6c1fd6799610247849abee843554c6fe9a8